### ISSUE TITLE
Does OpenMM works with any cuda drivers?

### ISSUE THREAD

**Issue by BlackPianoCat** [openmm/openmm#4706]
Recently, I have problems to run OpenMM in NVIDIA GeForce GTX graphic card. I use nvidia driver 535 on Debian Linux with CUDA 12.2. I also have tried to run it in our HPC cluster dgxs, which have NVIDIA A100 GPUs. In general, I was able to run OpenMM with GPU in the past, and particularly with CUDA driver, and I have tested that it is much faster. However, after some reinstallation of my operating system, where I was forced to change my drivers, the efficiency of the driver changed dramatically, and currently is comparable with CPU. 

Currently I am using pyenv environment and I installed OpenMM with `pip install OpenMM`. I am not sure if it is the best way to install it, but I am trying to avoid conda, because of inodes. However, I have tested conda installation and nothing changed.

Is it a known problem? Does OpenMM supports only specific types of drivers or I did not install it correctly?

**Comment by peastman** [comment]
It should work with all recent NVIDIA drivers.

> Currently I am using pyenv environment and I installed OpenMM with pip install OpenMM.

The current pip packages are experimental and shouldn't be relied on.  For the upcoming 8.2 release, we do plan to support pip.  There are instructions on how to install the current release candidate at https://github.com/openmm/openmm/issues/4659#issuecomment-2434177337.  Try that and see if it works.  You need to specify the package as `openmm[cuda12]` if you want it to include CUDA support.

**Comment by BlackPianoCat** [comment]
Thank you very much for your kind answer. Finally, I succeeded in running OpenMM with CUDA by updating my cuda drivers in Debian. I used the pip installation and it works fine for me now. It is sad news to hear that pip installation will stop to be supported, because it is much easier especially for HPC users who have problems with quotas. But anyway your team does great work with OpenMM, so the only thing I can do is to thank them for their valuable software.
